manager/testdata/new-blueprints/credit-card-detection-module.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: credit-card-detector
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+spec:
+  multi-user: false
+  chart:
+    name: ghcr.io/mesh-for-data/cc-detector:0.1.0
+  statusIndicators: # Indicators for a K8s deployment
+  kind: deployment # Assume for now that the program is a K8s deployment per dataset
+  capabilities:
+  - capability: audit
```

Comment:
This is a good point. This module might be special because it'll probably come from the policies. Output may have multiple options here. It could write logs to COS, send out mail, notify a slack channel, ...

Reply:
@froesef @simanadler I thought the decision was that the audit capability will not be a module but rather an external component that will receive events from the manager or analyze m4dapp status. Otherwise denials will not be recorded.